Package.swift

```diff
@@ -62,7 +62,7 @@ var dependencies: [Package.Dependency] {
                 from: "1.1.0"),
             .package(
                 url: "https://github.com/apple/swift-foundation-icu",
-                exact: "0.0.9"),
+                branch: "release/6.0"),
```

Comment:
Since this is being merged to the `main` branch, this should point at the `main` branch of `swift-foundation-icu` right?

Reply:
Ahhh good point! Thanks!